lib/src/platform_client.dart

```diff
@@ -247,6 +240,10 @@ class PlatformClient {
     return loginWithCredentials(credentials);
   }
 
+  /// Delete an account.
+  /// This action is irreversible and lead to permanent deletion of all data within the account.
+  Future<void> deleteAccount() => _httpDelete('/api/user/$_userId}');
```

Comment:
question: Is the bracket at the end of this desired?

Reply:
????!!! nop! Thanks!